TFT/src/User/API/menu.h

```diff
@@ -5,8 +5,8 @@
 extern "C" {
 #endif
 
-#include <stdint.h>
 #include <stdbool.h>
+#include <stdint.h>
```

Comment:
is there any special reason for this change?

Reply:
It seems only a wrong view update on git. the last pushed menu.h is as you suggested. thanks anyway